docs/advanced.md

```diff
@@ -362,3 +362,41 @@ is set to `None` or it cannot be inferred from it, HTTPX will default to
   ...
 }
 ```
+
+## SSL certificates
+
+!!! warning
+    This is an advanced section. You most likely won't need to use the features documented here, unless you know what you're doing.
+
+When making a request over HTTPS, HTTPX needs to verify the identity of the requested host. To do this, it uses a bundle of SSL certificates delivered by a trusted certificate authority (CA).
+
+By default, HTTPX uses the CA bundle provided by [Certifi](https://pypi.org/project/certifi/). This works fine in most cases, however in some advanced situations you may want to tweak this.
+
+### Using a custom CA bundle
```

Comment:
Is it CA bundle that we're passing or a certificate?

I'm not an expert by any means but I thought `pem` files are certificates?

Reply:
Yeah, I'm not sure on the wording myself. According to [this](https://serverfault.com/questions/200881/whats-a-ssl-certificate-bundle) a CA bundle is a group of root and intermediate certificates that work together… and I think `.pem` files can contain any number of certificates? Not sure, really. Ping @sethmlarson? 😄

Reply:
CA bundle is a file with one or more CA certificates concatenated together typically.  PEM can contain multiple certificates. (fun-fact: `certifi.where()` returns a .pem file)

Reply:
Thanks @sethmlarson. I pushed a commit based on this info and that in https://github.com/encode/httpx/issues/507#issuecomment-550371125 and elsewhere to more precisely explain in the developer interface what `verify` and `cert` are doing. :-)